frontend/src/constants/Routes.ts

```diff
@@ -22,6 +22,7 @@ export const CLASSROOMS_PAGE = "/teacher/classrooms";
 // Private Student Routes
 export const STUDENT_LANDING_PAGE = "/student";
 export const ASSESSMENT_SUMMARY_PAGE = "/student/assessment-summary";
+export const ASSESSMENT_EXPERIENCE_PAGE = "/student/assessment-experience";
```

Comment:
Was this URL specified by design? If not, I feel like the word "experience" could be confusing to a student, maybe they'd expect something like `/student/assessment/write`